src/main/java/com/epam/test_generator/entities/api/JwtTokenBuilder.java

```diff
@@ -2,7 +2,11 @@
 
 import com.auth0.jwt.JWT;
 import com.auth0.jwt.JWTCreator;
+import com.auth0.jwt.algorithms.Algorithm;
 import com.epam.test_generator.entities.User;
+import org.springframework.core.env.Environment;
+
+import java.io.UnsupportedEncodingException;
 
 public class JwtTokenBuilder {
```

Comment:
Let's also make methods non static. It will let us to mock instances of that class and inject dependencies if we need to.